Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.py

```diff
@@ -0,0 +1,602 @@
+        else:
+            return self.return_token_and_save_it_in_context(access_token_res)
+
+    def list_tenant_sites(self):
+        url = 'https://graph.microsoft.com/v1.0/sites'
+        query_string = {"search": "*"}
+        return self.http_call(method='GET', full_url=url, headers=self.headers, url_suffix='', params=query_string)
+
+    def list_drives_in_site(self, site_id=None, limit=None, next_page_url=None):
+        if not any([site_id, next_page_url]):
+            raise DemistoException('Please pass at least one argument to this command: \n'
+                                   'site_id: if you want to get all sites.\n'
+                                   'limit: if you want to limit the number of command\'s results. \n'
+                                   'next_page_url: if you have used the limit argument.')
+
+        if limit:
+            params = urlencode({'$top': limit})
+        else:
+            params = ''
+
+        if next_page_url:
+            url = url_validation(next_page_url)
+        else:
+            url = f'{BASE_URL}/sites/{site_id}/drives'
+
+        return self.http_call('GET', full_url=url, params=params, headers=self.headers, url_suffix='')
+
+    def list_drive_children(self, object_type, object_type_id, item_id=None, limit=None, next_page_url=None):
+        if next_page_url:
+            url = url_validation(next_page_url)
+        else:
+            if not item_id:
+                item_id = 'root'
+            if object_type == 'drives':
+                url = f'{object_type}/{object_type_id}/items/{item_id}/children'
+
+            elif object_type in ['groups', 'sites', 'users']:
+                url = f'{object_type}/{object_type_id}/drive/items/{item_id}/children'
+
+            url = BASE_URL + f'/{url}'
+
+        if limit:
+            params = urlencode({'$top': limit})
+        else:
+            params = ''
+
+        return self.http_call(method='GET', full_url=url, url_suffix='', params=params, headers=self.headers)
+
+    def replace_existing_file(self, object_type, object_type_id, item_id, entry_id):
+
+        file_path = demisto.getFilePath(entry_id).get('path', None)
+        if not file_path:
+            raise DemistoException(f'Could not find file path to the next entry id: {entry_id}. \n'
+                                   f'Please provide another one.')
+        if object_type == 'drives':
+            url = f'{object_type}/{object_type_id}/items/{item_id}/content'
+
+        elif object_type in ['groups', 'sites', 'users']:
+            url = f'{object_type}/{object_type_id}/drive/items/{item_id}/content'
+
+        # send request
+        url = BASE_URL + f'/{url}'
+        with open(file_path, 'rb') as file:
+            self.headers['Content-Type'] = 'application/octet-stream'
+            return self.http_call('PUT', full_url=url, data=file, headers=self.headers, url_suffix='')
+
+    def delete_file(self, object_type, object_type_id, item_id):
```

Comment:
this function needs documentation. see `get_encrypted()`(in your code) for example